docs/source/index.rst

```diff
@@ -3,13 +3,20 @@ PGMax Reference Documentation
 PGMax is a library for working with Factor Graphs in `JAX <https://jax.readthedocs.io/en/latest/>`_. It currently provides an interface for specifying factor graphs of any type, as well as an efficient implementation of max-product belief propagation and inference on these graphs.
 
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
```

Comment:
Is this the change that makes the API flatter?

Reply:
it makes the display on the table-of-contents flatter, but doesn't make the actual docs flatter. Once we get things to build with RTD, we can see if we don't like the hierarchy and flatten it if necessary!